lib/Sema/DerivedConformanceDistributedActor.cpp

```diff
@@ -571,6 +572,148 @@ deriveDistributedActorType_SerializationRequirement(
   return nullptr;
 }
 
+/******************************************************************************/
```

Comment:
- [ ] All of the synthesis code is pretty much 1:1 same as in an Actor, I'll refactor so we can share that code.

Reply:
Leaving this duplicated for now since we might want to change it a little bit actually for distributed -- pending discussion in evolution.

Reply:
(turns out having it separate was a good idea)